Wrappers/Python/ccpi/viewer/utils/conversion.py

```diff
@@ -946,29 +979,149 @@ def SetFileName(self, value):
     def GetFileName(self):
         return self.__FileName
 
-        
+
+class cilHDF5ImageResampleReader(cilBaseResampleReader):
```

Comment:
The only change to conversion.py that isn't just formatting is the addition of this `cilHDF5ImageResampleReader` class